modules/elasticsearch/variables.tf

```diff
@@ -138,11 +148,9 @@ variable "external_alb_setup" {
   description = "Should an external access be allowed to ES API on port 9201 with HTTPS and BasicAuth."
 }
 
-variable "elasticsearch_dns_ssl_name" {
-  default     = ""
-  description = "DNS name for Elasticsearch endpoint SSL. An SSL certificate is expected to be present in ACM for this domain. If left empty 'elasticsearch_dns_name' will be checked instead."
-}
-
+/* Same as `internal_alb`, except also requires a "certificate" variable, which is an
+   ARN for an ISSUED ACM certificate
+*/
```

Comment:
For these couple of doc blocks in `variables.tf`, I would recommend the following:
* move the previous comment (`Argument is expected to have this structure`), and the one noted above, to each variables `description` param
* use a heredoc to allow/make use of a multi-line comment.

In doing so, the TF registry will include the info (structure of the map) in the docs it auto-generates.

In addition to that, we could include a full example in the README.

Reply:
I was trying to think of a btter way to deal with a description for map type variables. I remember that terraform deprecated multiline description in variables, so I mistakenly assumed heredoc wouldn't work here either. Just tried it - works fine. Wondering though if it will be rendered properly in the tf registry, I guess we'll see. Changed.

As far as example goes, which README should we place it in one? `kibana`, `elasticsearch` or `elk-stack`, `elk-r53` or `web-alb module? All of them expect the same structure, so it be best to have a single place that explains it and that's where example would go too. Really wishing now terraform had types!!

Reply:
> As far as example goes, which README should we place it in one? kibana, elasticsearch or elk-stack, elk-r53 or web-alb module? All of them expect the same structure, so it be best to have a single place that explains it and that's where example would go too.

I guess the README for the `elk-stack` module could have the one big example?